content/docs/atom-guide/look-dev/six-point-lighting-tutorial.md

```diff
@@ -0,0 +1,439 @@
+   
+   3. Replace the `baseColorUv` parameter with `sixPointUv` in the call to `GetBaseColorInput()`.
+   ```glsl
+   float2 baseColorUv = uv[MaterialSrg::m_baseColorMapUvIndex];
+   float2 sixPointUv = GetUvForCurrentFrame(baseColorUv);
+   float3 sampledColor = GetBaseColorInput(MaterialSrg::m_baseColorMap, MaterialSrg::m_sampler, sixPointUv, MaterialSrg::m_baseColor.rgb, o_baseColor_useTexture);
+   float3 baseColor = BlendBaseColor(sampledColor, MaterialSrg::m_baseColor.rgb, MaterialSrg::m_baseColorFactor, o_baseColorTextureBlendMode, o_baseColor_useTexture);
+   ```
+
+3. Initialize the six-point surface properties.
+   
+   1. Find the *Specular* section.
+   
+   2. Set the six directional surface properties you added with the material inputs, according to the texture pack mode. You will need to handle both texture pack mode options and set the properties accordingly:
+   ```glsl
+   if(o_sixPointTexturePackMode == SixPointTexturePackMode::TpLftRtBt_FrBck)
+   {
+      float4 topLeftRightBottom = MaterialSrg::m_topLeftRightBottomMap.Sample(MaterialSrg::m_sampler, sixPointUv);
+      float4 frontBack = MaterialSrg::m_frontBackMap.Sample(MaterialSrg::m_sampler, sixPointUv);
+      surface.top = topLeftRightBottom.r;
+      surface.left = topLeftRightBottom.g;
+      surface.right = topLeftRightBottom.b;
+      surface.bottom = topLeftRightBottom.a;
+      surface.frontside = frontBack.r;
+      surface.backside = frontBack.g;
+   }
+   else
+   {
+      float4 rightLeftTop = MaterialSrg::m_rightLeftTopMap.Sample(MaterialSrg::m_sampler, sixPointUv);
+      float4 bottomBackFront = MaterialSrg::m_bottomBackFrontMap.Sample(MaterialSrg::m_sampler, sixPointUv);
+      surface.right = rightLeftTop.r;
+      surface.left = rightLeftTop.g;
+      surface.top = rightLeftTop.b;
+      surface.bottom = bottomBackFront.r;
+      surface.backside = bottomBackFront.g;
+      surface.frontside = bottomBackFront.b;
+   }
+   ```
+
+4. Initalize the `tangent` and `bitangent` surface properties right after:
+   ```glsl
+   surface.tangent = tangents[0];
+   surface.bitangent = bitangents[0];
+   ```
+
+## Add custom lighting
+Now that you have set up the six-point surface, you can use the new surface properties in custom lighting. You will make customize two types of lighting: directional lighting and image-based lighting (IBL). Directional lighting is lighting for if a light came from a direction, while IBL simulates omni-directional reflective, almost ambient, lighting from the environment around the entity.
+
+### Add custom directional lighting
+As discussed earlier, you will make a light map that uses the lighting direction to determine which combination of the six sides should be lit up. Then, you can take the intensity of the light and how much the texel should be lit up from lighting in that direction to compute the overall lighting on that particular texel.
```

Comment:
I agree, I mention "how much the texel should be lit up from lighting in that direction" a lot but it is very wordy.  I don't think there's an actual name for this since it's a niche idea, but at the beginning of the tutorial, I added

> {{< important >}}
> For simplicity in this tutorial, we will refer to "how much a texel should be lit up from lighting in that direction" as the "texel's directional intensity."
> {{< /important >}}

and then replaced the parts in the tutorial with "texel's directional intensity." Do you think this approach is better?

Reply:
How is this?

> As discussed earlier, you will make a light map that uses the lighting direction to determine which combination of the six sides to light up. Then, you will use the brightness with the texel's direction-respective lighting intensity to compute the overall lighting on that particular texel.